src/index.html

```diff
@@ -4,14 +4,208 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0,
+      minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      onsubmit="onSubmit()"
+      method="POST"
+    >
+      <fieldset class="fieldset">
+        <legend>
+          Personal information
+        </legend>
+        <div class="inputContainer">
+          <label for="Surname">
+            Surname:
+            <input
+              type="text"
+              name="Surname:"
+              autocomplete="off"
+              id="Surname"
+            >
+          </label> <br>
+        </div>
+        <div class="inputContainer">
+          <label for="Name">
+            Name:
+            <input
+              type="text"
+              name="Name:"
+              autocomplete="off"
+              id="Name"
+            >
+          </label> <br>
+        </div>
+        <div class="inputContainer">
+          <label for="Age">
+            How old are You?
+            <input type="number"
+              min="12"
+              max="100"
+              value="12"
+              id="Age"
+            >
+          </label> <br>
```

Comment:
Чи потрібен тобі `br` тег ?
```suggestion
          </label>
```